mongo/dbtest/db.go

```diff
@@ -180,31 +180,31 @@ func (dbs *DBServer) Stop() {
 	}
 }
 
-// Session returns a new session to the server. The returned session
+// Client returns a new client to the server. The returned client
 // must be closed after the test is done with it.
 //
-// The first Session obtained from a DBServer will start it.
-func (dbs *DBServer) Session() *mongo.Client {
+// The first Client obtained from a DBServer will start it.
+func (dbs *DBServer) Client() *mongo.Client {
```

Comment:
@alfrunes I am having second thoughts about that, this changes the interface, and will mean that we have to change one line per each unit test in every service extra. @kjaskiewiczz what do you think? If there is someone else who thinks that this change of exported func is a good idea I will not argue.

Reply:
It is definitely bad practice if we let it stick in my opinion.

Reply:
However, I can rewrite the comment as I literally just did a regex replace over the whole library...

Reply:
since Marcin approved I have no problem with that patch. go ahead.